.gitignore

```diff
@@ -22,3 +22,4 @@ bin
 *.swp
 *.swo
 *~
+output
```

Comment:
Add line in the end

Reply:
> Add line in the end

Maybe I should add the script to fix this kind of problem, and it'll be more useful after the CI is ready.